### PR TITLE
Toggle

### PR DESCRIPTION
**Pull request by karlesh**
# Alaska Airlines Pull Request

## Summary:

We changed 'toggleViewable' function to fire the toggle event. 
This allows users of auro-dialog to run events when the dialog is closed using the close button. 

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: src/auro-dialog.js [hunk 1]
```diff
@@ -102,8 +106,8 @@ class AuroDialog extends LitElement {
               <slot name="header">Default header ...</slot>
             </h1>
             ${this.modal
-              ? html``
-              : html`
+        ? html``
```
Comment (blackfalcon) [inline src/auro-dialog.js:109]: What is happening here?

**Comment by blackfalcon** [comment]
@Kiricon can you please better describe the problem you are solving for? Why the change?

**Review by blackfalcon — DISMISSED** [review]

**Comment by blackfalcon** [comment]
Added @gusnaughton as a reviewer

**Review by gusnaughton — APPROVED** [review]

**Review by gusnaughton — APPROVED** [review]

**Review by gusnaughton — APPROVED** [review]

**Pull request merged**